scripts/encoding_pathway_analysis.R

```diff
@@ -61,54 +85,84 @@ for (hgsc_file in hgsc_files) {
   direction <- hgsc_info[3]
   direction <- unlist(strsplit(direction, "[.]"))[1]
   base_name <- paste(node, direction, sep = "_")
-  
+
   # Read in hgsc genes file
-  hgsc_node_df <- readr::read_tsv(hgsc_file)
+  hgsc_node_df <- readr::read_tsv(
+    hgsc_file,
+    col_types = readr::cols(genes = readr::col_character(),
+                            weight = readr::col_double())
+    )
+
   hgsc_genes <- hgsc_node_df$genes
 
   # Perform pathway analysis with WebGestaltR
-  g <- RunWebGestalt(hgsc_genes, base_name, hgsc_out_dir, bg_file)
+  g <- RunWebGestalt(genes = hgsc_genes,
```

Comment:
I'm not too familiar with WebGestalt.  Is this something that will continuously need to be updated in order for someone to rerun your code?

Reply:
WebGestlat provides a nice interface for pathway analysis

But yes :frowning: need to update sometimes. a dependency that we should move away from when possible